Description: Fix bktr detection for newer kFreeBSD systems
 Newer systems have their headers in multiarch paths, so we must update motion
 configure scripts to use these. Otherwise, it fails to detect bktr and tries
 to use machine/ioctl_*.h instead, breaking the build.
Author: Steven Chamberlain <steven@pyro.eu.org>
Bug-Debian: https://bugs.debian.org/817757
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/configure.ac
+++ b/configure.ac
@@ -89,7 +89,7 @@
 
 	AC_MSG_CHECKING(bktr headers in /usr/include/dev/bktr)
 
-	if test -f /usr/include/dev/bktr/ioctl_meteor.h && test -f /usr/include/dev/bktr/ioctl_bt848.h; then
+	if test -f /usr/include/*/dev/bktr/ioctl_meteor.h && test -f /usr/include/*/dev/bktr/ioctl_bt848.h; then
 		AC_MSG_RESULT(yes)
 	else
 		AC_MSG_RESULT(no)
